src/components/Publications.jsx

```diff
@@ -1,45 +1,90 @@
 import '../styles/Publications.css'
+
 import img1 from '../assets/img1.gif'
+
 import img2 from '../assets/img2.jpg'
+
 import img3 from '../assets/img3.jpg'
+
 import img4 from '../assets/img4.jpg'
+
 import img5 from '../assets/Untitled.png'
 
+
+
 function Publications() {
+
     return (
+
         <>
+
             <div className='publication-title'><h2>Selected Publications</h2></div>
```

Comment:
Put className directly on the h2 element?
```suggestion
            <h2 className='publication-title'>Selected Publications</h2>
```